run.py

```diff
@@ -787,6 +787,9 @@ def fetch_test_details(var) -> dict:
             if skip_subscription is True:
                 config["skip_subscription"] = True
 
+            if config.get("skip_version_compare"):
```

Comment:
can you please re-consider having cli option for version comparison Since, 
- It is already handled and decided at test case level.
```
module: test_ansible_upgrade.py
      config:
        build: '4.x'
        skip_version_compare: true
```
- It's only used in 4x upgrade test module.